agent/app/agent_capability_windows.go

```diff
@@ -24,12 +24,27 @@ import (
 )
 
 var (
-	capabilityDepsRootDir          = filepath.Join(config.AmazonECSProgramFiles, "managed-agents")
+	capabilityDepsRootDir  = filepath.Join(config.AmazonECSProgramFiles, "managed-agents")
+	ssmPluginDir           = filepath.Join(config.AmazonProgramFiles, "SSM", "Plugins")
+	sessionManagerShellDir = filepath.Join(ssmPluginDir, "SessionManagerShell")
+	awsCloudWatchDir       = filepath.Join(ssmPluginDir, "awsCloudWatch")
```

Comment:
how is this dir and below one used for exec feature?

Reply:
The directories are required by the SSM agent which runs inside the container. The SessionManagerShell is for the ssm start-session, and the other 2 are for if the customer decides to push their logs to S3 or cloud watch